### PR TITLE
Update manifest: Nushell.Nushell version 0.100.0

### LINKED ISSUE #200894: [Package Issue]: Nushell.Nushell Installation doesn't respect the selected scope
### Please confirm these before moving forward

- [x] I have searched for my issue and not found a work-in-progress/duplicate/resolved issue.
- [x] I have not been informed if the issue is resolved in a preview version of the winget client.

### Category of the issue

Installation issue.

### Brief description of your issue

Both `user` and `machine` scopes behave in misleading way during installation. Package is either not installed or installed to the wrong location.

### Steps to reproduce

- 
  ```
  winget install -e --id Nushell.Nushell --scope user
  ```
  See the error message.
<br>     
  
- ```
  winget install -e --id Nushell.Nushell --scope machine
  ```
  Check the actual install location.


### Actual behavior

- Installing to `user` scope:
  ```
  winget install -e --id Nushell.Nushell --scope user
  ```
  ```
  No applicable installer found; see logs for more details.
  ```

- Installing to `machine` scope as a regular user:
  ```
  winget install -e --id Nushell.Nushell --scope machine
  ```
  ```
  Failed to open the predefined source; please report to winget maintainers.
  An unexpected error occurred while executing the command:
  0x80070005 : Access is denied.
  ```
- Installing to `machine` scope as administrator:
  ```
  winget install -e --id Nushell.Nushell --scope machine
  ```
  Package is installed per user `(%LOCALAPPDATA%\Programs\nu)`


### Expected behavior

1. Package installation respects the `--scope` flag.
2. Package is not installed to unexpected locations.

### Environment

```raw
Windows Package Manager (Preview) v1.10.40-preview
Copyright (c) Microsoft Corporation. All rights reserved.

Windows: Windows.Desktop v10.0.19045.5247
System Architecture: X64
Package: Microsoft.DesktopAppInstaller v1.25.40.0

Winget Directories
-------------------------------------------------------------------------------------------------------------------------------
Logs                               %LOCALAPPDATA%\Packages\Microsoft.DesktopAppInstaller_8wekyb3d8bbwe\LocalState\DiagOutputDir
User Settings                      %LOCALAPPDATA%\Packages\Microsoft.DesktopAppInstaller_8wekyb3d8bbwe\LocalState\settings.json
Portable Links Directory (User)    %LOCALAPPDATA%\Microsoft\WinGet\Links
Portable Links Directory (Machine) C:\Program Files\WinGet\Links
Portable Package Root (User)       %LOCALAPPDATA%\Microsoft\WinGet\Packages
Portable Package Root              C:\Program Files\WinGet\Packages
Portable Package Root (x86)        C:\Program Files (x86)\WinGet\Packages
Installer Downloads                %USERPROFILE%\Downloads

Links
---------------------------------------------------------------------------
Privacy Statement   https://aka.ms/winget-privacy
License Agreement   https://aka.ms/winget-license
Third Party Notices https://aka.ms/winget-3rdPartyNotice
Homepage            https://aka.ms/winget
Windows Store Terms https://www.microsoft.com/en-us/storedocs/terms-of-sale

Admin Setting                             State
--------------------------------------------------
LocalManifestFiles                        Disabled
BypassCertificatePinningForMicrosoftStore Disabled
InstallerHashOverride                     Disabled
LocalArchiveMalwareScanOverride           Disabled
ProxyCommandLineOptions                   Disabled
DefaultProxy                              Disabled
```

### Screenshots and Logs

_No response_

### PR DESCRIPTION
**Pull request by Dragon1573**
## What's changed

- Remove `Scope` definition in top-level properties.
- Duplicate installer entrance in `Installers` properties.
- Explicitly add `Scope` definition for each entrance.

## Checklist for Pull Requests

- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Resolve #200894

## Manifests

- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.9 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.9.0)?

Note: `<path>` is the directory's name containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/200911)

### CODE REVIEW

**Review by Trenly — CHANGES_REQUESTED** [review]
This is a wix installer and as such does not support user scope - it will always write the ARP entries to machine scope

File: manifests/n/Nushell/Nushell/0.100.0/Nushell.Nushell.installer.yaml [hunk 1]
```diff
@@ -18,10 +17,22 @@ AppsAndFeaturesEntries:
...
 - Architecture: x64
+  Scope: machine
   InstallerUrl: https://github.com/nushell/nushell/releases/download/0.100.0/nu-0.100.0-x86_64-pc-windows-msvc.msi
   InstallerSha256: 235DC70E1D63F2B162754A88376D9AAD5B061CD292E0D989D562D52E35DD818A
   ProductCode: '{250531A9-1797-4327-91C2-C442165F61D8}'
+- Architecture: x64
+  Scope: user
+  InstallerUrl: https://github.com/nushell/nushell/releases/download/0.100.0/nu-0.100.0-x86_64-pc-windows-msvc.msi
+  InstallerSha256: 235DC70E1D63F2B162754A88376D9AAD5B061CD292E0D989D562D52E35DD818A
+  ProductCode: '{250531A9-1797-4327-91C2-C442165F61D8}'
```
Comment (Dragon1573) [inline manifests/n/Nushell/Nushell/0.100.0/Nushell.Nushell.installer.yaml:28]:
```suggestion
```

File: manifests/n/Nushell/Nushell/0.100.0/Nushell.Nushell.installer.yaml [hunk 2]
```diff
@@ -18,10 +17,22 @@ AppsAndFeaturesEntries:
...
+- Architecture: arm64
+  Scope: machine
+  InstallerUrl: https://github.com/nushell/nushell/releases/download/0.100.0/nu-0.100.0-aarch64-pc-windows-msvc.msi
+  InstallerSha256: 6BD09D62D518C62E527C3D05D0C55B16C0F6DD743654EFF1EF9E34F00826F4EA
+  ProductCode: '{7002B193-9A8E-405E-A866-451E38FF69D4}'
 - Architecture: arm64
+  Scope: user
   InstallerUrl: https://github.com/nushell/nushell/releases/download/0.100.0/nu-0.100.0-aarch64-pc-windows-msvc.msi
   InstallerSha256: 6BD09D62D518C62E527C3D05D0C55B16C0F6DD743654EFF1EF9E34F00826F4EA
   ProductCode: '{7002B193-9A8E-405E-A866-451E38FF69D4}'
```
Comment (Dragon1573) [inline manifests/n/Nushell/Nushell/0.100.0/Nushell.Nushell.installer.yaml:38]:
```suggestion
```

**Review by Dragon1573 — COMMENTED** [review]
User scope should be removed from the manifest.

**Review by Trenly — CHANGES_REQUESTED** [review]
And now the PR won’t fix the bug that it's linked to! Moving the scope to each individual installer will have no effect; The cleanest solution here is to make the scope undefined by removing it from the manifest entirely

**Review by stephengillie — APPROVED** [review]

**Pull request merged**